app/http.php

```diff
@@ -302,6 +303,60 @@ function createDatabase(App $app, string $resourceKey, string $dbName, array $co
 
                 $dbForPlatform->createCollection('bucket_' . $bucket->getInternalId(), $attributes, $indexes);
             }
+
+            if ($dbForPlatform->getDocument('buckets', 'screenshots')->isEmpty() &&
+                !$dbForPlatform->exists($dbForPlatform->getDatabase(), 'bucket_2')) {
+                Console::info("    └── Creating screenshots bucket...");
+                $dbForPlatform->createDocument('buckets', new Document([
+                    '$id' => ID::custom('screenshots'),
+                    '$collection' => ID::custom('buckets'),
+                    'name' => 'Screenshots',
+                    'maximumFileSize' => (int) System::getEnv('_APP_STORAGE_LIMIT', 0), // 10MB
```

Comment:
Not sure I would use the server default here, can cause weird side effects if too small

Reply:
Will hard-code to 5MB, should be more than enough.